app/controllers/api/configuration_script_sources_controller.rb

```diff
@@ -18,10 +18,30 @@ def delete_resource(type, id, _data = {})
       action_result(false, err.to_s)
     end
 
+    def create_resource(_type, _id, data)
+      validate_attrs(data)
+      manager_id = parse_id(data['manager_resource'], :providers)
```

Comment:
Can you add a check here before we do a resource_search,

```
raise "Must specify a valid manager_resource href or id" unless manager_id
```
 and add a test for that bad_request, maybe passing in a bogus /api/users/10 href.  Thanks.